packages/central-server/src/apiV2/tasks/CreateTask.js

```diff
@@ -21,6 +21,16 @@ export class CreateTask extends CreateHandler {
   }
 
   async createRecord() {
-    await this.insertRecord();
+    const { comment, ...data } = this.newRecordData;
+
+    return this.models.wrapInTransaction(async transactingModels => {
+      const task = await transactingModels.task.create(data);
+      if (comment) {
+        await task.addComment(comment, this.req.user.id);
```

Comment:
I decided to handle adding comments in the central server because I don't think that anyone will ever add a comment outside of this context (or on the model)

Reply:
I am often unsure about where to add things like this too but I agree that central-server is a good choice. I think that it is probably the best starting place for most back end functionality unless there is a good reason not to put it here.